ProcessQueue/function.json

```diff
@@ -0,0 +1,12 @@
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "name": "msg",
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "new-scanreports",
```

Comment:
change queue name and storage name, and need to set up said storage/queue in Azure

Reply:
queue: `scanreports`, storage: `ccom-storage-dev`

Reply:
Update: `ccomstoragedev`